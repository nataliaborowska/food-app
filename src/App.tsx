import React from 'react';
import {BrowserRouter as Router, Route} from 'react-router-dom';
import 'bootstrap/dist/css/bootstrap.min.css';

import {Navbar} from './modules/Navbar';
import {Dishes} from './modules/Dishes';
import {Drinks} from './modules/Drinks';
import {ErrorBoundary} from './common/ErrorBoundary';
import {Home} from './modules/Home';
import styles from './App.module.scss';

const App: React.SFC = () => {
  return (
    <Router>
      <div className='container'>
        <Navbar />
        <ErrorBoundary><Route path="/" exact component={Home} /></ErrorBoundary>
        <ErrorBoundary><Route path="/dishes" exact component={Dishes} /></ErrorBoundary>
        <ErrorBoundary><Route path="/drinks" exact component={Drinks} /></ErrorBoundary>
      </div>
    </Router>
  )
}

export default App;
