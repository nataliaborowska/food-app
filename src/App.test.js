import {shallow} from 'Enzyme';

import App from './App';

it('renders App component', () => {
  const wrapper = shallow(<App />);

  expect(wrapper).toMatchSnapshot();
});
