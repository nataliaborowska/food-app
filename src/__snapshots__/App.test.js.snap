// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component 1`] = `
<BrowserRouter>
  <div
    className="container"
  >
    <Navbar />
    <ErrorBoundary>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
    </ErrorBoundary>
    <ErrorBoundary>
      <Route
        component={[Function]}
        exact={true}
        path="/dishes"
      />
    </ErrorBoundary>
    <ErrorBoundary>
      <Route
        component={[Function]}
        exact={true}
        path="/drinks"
      />
    </ErrorBoundary>
  </div>
</BrowserRouter>
`;
